### PR TITLE
Remove file button added

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Fixes: #429 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

editor7

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by NitinBhasneria** [comment]
@VladimirMikulic @nstjean @cesswairimu Have a look.

**Comment by VladimirMikulic** [comment]
What if I click the button during the upload process? You need to handle all possible usecases.

**Comment by NitinBhasneria** [comment]
@VladimirMikulic What is expected in this case?

**Comment by VladimirMikulic** [comment]
Cancelling an upload, I guess?

**Comment by NitinBhasneria** [comment]
Currently, the button will not cancel the process.

**Comment by NitinBhasneria** [comment]
It will be easy to cancel the process by clicking. But we should discuss this I guess. 
Let's pin @cesswairimu @nstjean @jywarren.
Thanks @VladimirMikulic

**Comment by nstjean** [comment]
I think it should cancel the upload in that situation.

**Comment by jywarren** [comment]
Agree! This is cool!!!

**Comment by NitinBhasneria** [comment]
editor1
The remove file button will work in cancelling the upload too. Have a look @jywarren @VladimirMikulic @nstjean

**Comment by VladimirMikulic** [comment]
@NitinBhasneria this is awesome. Now, if you could just fix styling, that would be great!

**Comment by NitinBhasneria** [comment]
@VladimirMikulic Styling will be automatically fixed after PR #448. So that both buttons will have the same styling thanks

**Comment by NitinBhasneria** [comment]
This PR is ready for the merge for now. Thanks

**Comment by NitinBhasneria** [comment]
editor
Styling done.
Have a look @VladimirMikulic Thanks.

**Comment by NitinBhasneria** [comment]
It's changed and made consistent in #462. Also, this PR is ready for merge. Thanks
 Have a look. @jywarren @VladimirMikulic @govindgoel @nstjean

**Comment by VladimirMikulic** [comment]
Screenshot of your change?

**Comment by NitinBhasneria** [comment]
You can look it in PR #463. @VladimirMikulic Thanks

**Comment by VladimirMikulic** [comment]
@NitinBhasneria they are not consistent.

nitin-btns

**Comment by NitinBhasneria** [comment]
Actually, this PR will have remove button styling and working. The choose file button styling will be different we cant just style it and so `label` is added which is reflected in PR #463 
Thanks.

**Comment by NitinBhasneria** [comment]
Together they will have a look like this. @VladimirMikulic 
editor

**Comment by VladimirMikulic** [comment]
Alright, but please brighten the black colour a bit. #000 is almost never used!

**Comment by NitinBhasneria** [comment]
Done for the colour @VladimirMikulic

**Comment by NitinBhasneria** [comment]
@jywarren its ready. Please jave a look

**Comment by NitinBhasneria** [comment]
@VladimirMikulic please approve this.

**Comment by NitinBhasneria** [comment]
@jywarren this is ready. Thanks

**Comment by NitinBhasneria** [comment]
@VladimirMikulic The last screenshot is the final style. Is there any doubt?

**Comment by VladimirMikulic** [comment]
I was refering to code style, hence quotes.

**Comment by NitinBhasneria** [comment]
Is there something wrong with my code🙄?

File: src/modules/PublicLab.MainImageModule.js [hunk 1]
```diff
@@ -42,106 +43,116 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+                ).attr('aria-valuenow', '100')
 
-      }
+            }
 
-    });
+        });
 
+        // Remove Image button
+        var mainImage = document.getElementById("mainImage");
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:148]: Quotes inconsistencies.

File: src/modules/PublicLab.MainImageModule.js [hunk 2]
```diff
@@ -42,106 +43,116 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
-      progressall: function (e, data) {
+            progressall: function(e, data) {
 
-        var progress = parseInt(data.loaded / data.total * 100, 10);
-        _module.el.find('.progress .progress-bar').css(
-          'width',
-          progress + '%'
-        ).attr('aria-valuenow', '100')
+                var progress = parseInt(data.loaded / data.total * 100, 10);
+                _module.el.find('.progress .progress-bar').css(
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:138]: Quotes inconsistencies.

**Comment by NitinBhasneria** [comment]
@VladimirMikulic have alook. Thanks

**Pull request closed without merging**